### PR TITLE
Add option to Load yang models from class path inside gNMI SB - Master

### PR DESCRIPTION
**Pull request by PeterSuna**
Implement option to load Yang models from class path with initialYangModels parameter in gNMI SB configuration. Both option for loading models from class path or from folder containing models can be used at the same time. Loading YANG models from class path resolve openconfig models duplicity inside test resources.

### CODE REVIEW

**Review by vainyksi — CHANGES_REQUESTED** [review]

File: lighty-applications/lighty-rcgnmi-app-aggregator/README.md [hunk 1]
```diff
@@ -59,10 +59,16 @@ Both of these options will load the YANG files into data-store, from which **lig
...
           "INITIAL_FOLDER_PATH"
+        ],
+        "initialYangModels": [
+            { "nameSpace":"MODEL_NAMESPACE","name":"MODEL_NAME","revision":"MODEL_REVISION"}
         ]
       }
     ```
-    3. Change `INITIAL_FOLDER_PATH`, from the JSON block above, to a folder path, which contains YANG models you wish to load into the datastore. These models will be then automatically loaded on startup.
+    3. Use one or both [`initialYangsPaths`,`initialYangModels`] option for adding YANG models to gNMI module. 
+       1. `initialYangsPaths`: Change `INITIAL_FOLDER_PATH`, from the JSON block above, to a folder path, which contains YANG models you wish to load into the datastore.
```
[the remark this answers is not in the document]
Comment (PeterSuna) [reply lighty-applications/lighty-rcgnmi-app-aggregator/README.md]: done

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app/pom.xml [hunk 2]
```diff
@@ -62,6 +62,10 @@
             <groupId>com.github.spotbugs</groupId>
             <artifactId>spotbugs-annotations</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.lighty.models</groupId>
+            <artifactId>openconfig-models</artifactId>
+        </dependency>
```
[the remark this answers is not in the document]
Comment (PeterSuna) [reply lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app/pom.xml]: done

File: lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/test/java/io/lighty/gnmi/southbound/mountpoint/codecs/testcases/CodecTestCasesBase.java [hunk 3]
```diff
@@ -39,18 +43,23 @@
...
-    protected static final String ETH_ID = "openconfig-if-ethernet";
-    protected static final String VLAN_ID = "openconfig-vlan";
-    protected static final String PLT_ID = "openconfig-platform";
+    private static final String BASE_YANGS_PATH = "src/test/resources/additional/test/schema";
+    private static final String OPENCONFIG_GNMI_CONFIG = "/lightyconfigs/openconfig_gnmi_config.json";
+    private static final String IT_TYPES_ID = "openconfig-if-types";
+    private static final String VLAN_ID = "openconfig-vlan";
+    private static final String PLT_ID = "openconfig-platform";
+    static final String ETH_ID = "openconfig-if-ethernet";
+    static final String IT_ID = "openconfig-interfaces";
```
[the remark this answers is not in the document]
Comment (PeterSuna) [reply lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/test/java/io/lighty/gnmi/southbound/mountpoint/codecs/testcases/CodecTestCasesBase.java]: done

File: lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/main/java/io/lighty/gnmi/southbound/schema/loader/util/YangLoadModelUtil.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import org.apache.commons.io.IOUtils;
+import org.opendaylight.yangtools.concepts.SemVer;
+import org.opendaylight.yangtools.yang.common.Revision;
+import org.opendaylight.yangtools.yang.model.repo.api.YangTextSchemaSource;
+import org.opendaylight.yangtools.yang.parser.api.YangSyntaxErrorException;
+import org.opendaylight.yangtools.yang.parser.rfc7950.repo.YangModelDependencyInfo;
+
+public class YangLoadModelUtil {
```
[the remark this answers is not in the document]
Comment (PeterSuna) [reply lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/main/java/io/lighty/gnmi/southbound/schema/loader/util/YangLoadModelUtil.java:21]: YangLoadModelUtil  is called by both ByClassPathYangLoaderService and ByPathYangLoaderService. So this class is tested whenever is called load method in YangLoaderService.
E.q:
https://github.com/PANTHEONtech/lighty/pull/856/files#diff-c4ac418d67e7038568821d00cc4bc5573ed914192bac85d33079d3a6fa270bf9R43

**Review by vainyksi — APPROVED** [review]

**Pull request merged**